plugins/CustomVariables/Reports/GetCustomVariables.php

```diff
@@ -34,5 +34,39 @@ public function configureView(ViewDataTable $view)
         $view->config->addTranslation('label', Piwik::translate('CustomVariables_ColumnCustomVariableName'));
         $view->requestConfig->filter_sort_column = 'nb_actions';
         $view->requestConfig->filter_sort_order  = 'desc';
+
+        if($this->isReportContainsUnsetVisitsColumns()) {
```

Comment:
I presume normal way would be something like 
```php
$this->config->filters[] = function (DataTable $table) use ($view) {
    // logic
   $message = ...
   $view->config->show_footer_message = $message
}
```

to prevent fetching report twice